ui/components/SidebarComponent.tsx

```diff
@@ -21,33 +22,63 @@ const centerFlexStyle = {
   marginBottom: '0.5rem',
 };
 
-export default function SidebarComponent(props: {}) {
+export default function SidebarComponent() {
   const timezones = ['UTC', 'Local', 'Relative'];
   const [zone, setZone] = useLocalStorage('timezone-ui', '');
 
   const {
     data: version,
-    error,
     isLoading,
   }: { data: UVersionResponse; error: any; isLoading: boolean } = useSWR(
     '/api/version',
     fetcher
   );
 
+  const [sidebarState, setSidebarState] = useLocalStorage(
+    'peerdb-sidebar',
+    'open'
+  );
   return (
     <SessionProvider>
       <Sidebar
+        style={{ width: sidebarState == 'closed' ? 'fit-content' : 'auto' }}
         topTitle={
-          <Label as={Link} href='/'>
-            <div className='cursor-pointer'>
-              <BrandLogo />
-            </div>
-          </Label>
+          <div
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'space-between',
+              width: '100%',
+            }}
+          >
+            {sidebarState == 'closed' ? (
+              <></>
+            ) : (
+              <Label as={Link} href='/'>
+                <div className='cursor-pointer'>
+                  <BrandLogo />
+                </div>
+              </Label>
+            )}
```

Comment:
```suggestion
            {sidebarState === 'open' && (
              <Label as={Link} href='/'>
                <div className='cursor-pointer'>
                  <BrandLogo />
                </div>
              </Label>
            )}
```

Reply:
Sidebar component's attributes don't accept `boolean | JSXElement` which is the resulting type of condition && element